media/js/firefox/onboarding/fxa-complex.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($) {
```

Comment:
Yes indeed. I put such a task in our last sprint planning spreadsheet (and brought it up during the planning session), but did not yet file a bug. Will do that now.